src/lib/mongoBackend/mongoGetSubscriptions.h

```diff
@@ -37,7 +37,7 @@
 *
 * mongoListSubscriptions -
 */
-void mongoListSubscriptions
+extern void mongoListSubscriptions
```

Comment:
Suggestion: why not call the routine mongoGetSubscriptions instead of mongoListSubscriptions?

NTC